.ci/os.ps1

```diff
@@ -5,10 +5,9 @@ If (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdent
     exit
 }
 
-# CI configures these, uncoment if running manually 
-#
-# $env:ES_BUILD_JAVA="java12" 
-#$env:ES_RUNTIME_JAVA="java11" 
+$AppProps = ConvertFrom-StringData (Get-Content .ci/java-versions.properties -raw) 
+$env:ES_BUILD_JAVA=$AppProps.ES_BUILD_JAVA
```

Comment:
Is there some generic way to just expose these environment variables so we don't have to hard code each of them? For example, loop over `$AppProps` and set each as a corresponding environment variable so we don't have to touch this script should we add something new to `java-versions.properties`?

Reply:
I considered it, but don't really see a need to make this very generic. 
This has the advantage of being simple and readable. I suggest we change ti if we'll have to add one more.